index.html

```diff
@@ -5,29 +5,24 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Flexbox</title>
+    <title>Vitor Guedes Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <header>
-        <div class="container">
-            <div class="logobox">
-                <a class="logo menupart neonText2" href="#">My<strong>Logo</strong></a>
-            </div>
+    <header class="container">
+        <a class="logo neonText2" onclick="closeMenu()" href="#">My<strong>Vitor</strong></a>
     
-            <nav class="menu">
-                <div class="hoverlogo">
-                    <a class="menupart menupartlogo" href="#"><img src="images/menupequeno.png" alt=""></a>
-                </div>
```

Comment:
- [ ] Please reduce the right margin of your hamburger icon a little from the right. 👇🏾 
your version
![Screenshot (117)](https://user-images.githubusercontent.com/65298048/173617352-1d99865b-e9bc-4fe6-a777-37adcf8699d2.png)
figma..
![Screenshot (114)](https://user-images.githubusercontent.com/65298048/173617584-1d2def6d-a748-402e-8514-f9f12da9bca1.png)